client/master.go

```diff
@@ -571,17 +573,28 @@ func (m *masterClient) RegisterPartition(ctx context.Context, partition *entity.
 }
 
 //send HTTPPost request
-func (m *masterClient) HTTPPost(url string, reqBody string) (response []byte, e error) {
+func (m *masterClient) HTTPPost(ctx context.Context, url string, reqBody string) (response []byte, e error) {
 	//process panic
 	defer func() {
 		if info := recover(); info != nil {
 			e = fmt.Errorf("panic is %v", info)
 		}
 	}()
+	var err error
+	if config.Conf().Global.MergeRouter {
+		config.Conf().Router.RouterIPS, err = m.QueryRouter(ctx, config.Conf().Global.Name)
+		if err != nil {
+			return nil, fmt.Errorf("query router err: %v", err)
+		}
+	}
+	query := netutil.NewQuery().SetHeader(Authorization, util.AuthEncrypt(Root, m.cfg.Global.Signkey))
+	query.SetMethod(http.MethodPost)
+	query.SetUrlPath(url)
+	query.SetReqBody(reqBody)
+	query.SetContentTypeJson()
+	query.SetTimeout(60)
+	num := 0
```

Comment:
This method requires looping through all available nodes. When a node fails, the processing logic of other nodes can also be found.